# std_msgs/Header 
Header header

# Motor 1 & 2 encoder values
int32 m1_enc_val
int32 m2_enc_val

# Motor 1 & 2 speed values in QPPS (+/-)
int32 m1_enc_qpps
int32 m2_enc_qpps
